src/D2L.Security.OAuth2/D2L.Security.OAuth2.csproj

```diff
@@ -19,6 +19,7 @@
 		<PackageReference Include="D2L.Services.Core.Exceptions" />
 		<PackageReference Include="System.Collections.Immutable" />
 		<PackageReference Include="System.IdentityModel.Tokens.Jwt" />
+		<PackageReference Include="System.Net.Http" />

```

Comment:
This is an unmaintained package (last published 2018), its part of the standard library now.

This library multi-targets .NET 4.7.2 and .NET 5 and runs in Linux on .NET 6 just fine.